app/src/main/res/layout/fragment_filter_list.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
```

Comment:
Please add corresponding `Start`/`End` for `Left`/`Right` fields where appopriate, .e.g. `android:gravity:start` for `android:gravity="left` and so on.

Reply:
I'm not all that expert at android layouts, but I believe that I wanted everything there to be gravity:start to support RTL languages (even though we don't have string support for any atm?). Could you please take a look? Thanks!

Reply:
Yes, that's right, for now there's no supported RTL language. Android Studio should highlight all occurances where `end` or `start? could be added or replace.